lib/validate.ts

```diff
@@ -0,0 +1,12 @@
+import { VideoCodec } from 'livekit-client';
+
+export function validateVideoCodec(codec: string | undefined | null): VideoCodec | undefined {
+  if (codec === 'vp8' || codec === 'vp9' || codec === 'h264' || codec === 'av1') {
```

Comment:
same as above, we can use the exported `videoCodecs` array instead for this check.